docs/user_guide/mitigation/preprocessing.rst

```diff
@@ -20,23 +20,43 @@ Sensitive features can be correlated with non-sensitive features in the dataset.
 By applying the :code:`CorrelationRemover`, these correlations are projected away
 while details from the original data are retained as much as possible (as measured
 by the least-squares error). The user can control the level of projection via the
-:code:`alpha` parameter. In mathematical terms, assume we have the original dataset
-:math:`X` which contains a set of sensitive attributes :math:`S` and a set of
-non-sensitive attributes :math:`Z`. The removal of correlation is then
-described as:
+:code:`alpha` parameter.
+
+In mathematical terms, assume we have the original dataset :math:`X`, which
+contains a set of **sensitive attributes** denoted by :math:`S` and a set of
+**non-sensitive attributes** denoted by :math:`Z`. The goal is to remove
+correlations between the sensitive attributes and the non-sensitive attributes.
+
+Let :math:`m_s` and :math:`m_{ns}` denote the number of sensitive and non-sensitive
+features, respectively.
+Let :math:`\bar{S}` represent the mean of the sensitive attributes, *i.e.*,
+:math:`\bar{S} = (\bar{s}_1, \dots, \bar{s}_{m_s})`, where
+:math:`\bar{s}_j` is the mean of the :math:`j\text{-th}` sensitive feature.
+
+For each non-sensitive feature :math:`z_j` in :math:`Z`,
+we compute an optimal weight vector :math:`w_j^* \in \mathbb{R}^{m_s}` that minimizes the
+following least squares objective:
 
 .. math::
-        \min _{\mathbf{z}_{1}, \ldots, \mathbf{z}_{n}} \sum_{i=1}^{n}\left\|\mathbf{z}_{i}
-        -\mathbf{x}_{i}\right\|^{2} \\
-        \text{subject to} \\
-        \frac{1}{n} \sum_{i=1}^{n} \mathbf{z}_{i}\left(\mathbf{s}_{i}-\overline{\mathbf{s}}
-        \right)^{T}=\mathbf{0}
-
-The solution to this problem is found by centering sensitive features, fitting a
-linear regression model to the non-sensitive features and reporting the residual.
-The columns in :math:`S` will be dropped from the dataset :math:`X`.
-The amount of correlation that is removed can be controlled using the
-:code:`alpha` parameter. This is described as follows:
+
+    \min _{w} \| z_j - (S - \bar{S}) w \|_2^2
```

Comment:
In numpy, this works thanks to broadcasting, but here we have a dimension mismatch, so we would need to write `\mathbf{S}-\mathbf{1}\bar{\mathbf{s}}^\top` where `\mathbf{1}` is the all-ones vector in `\mathbb{R}^n`.